### ISSUE TITLE
qt6ct makes all Qt Quick applications crash

### ISSUE THREAD

**Issue by ec1oud** [trialuser02/qt6ct#32]
This is pretty bad, and makes Qt look bad.  We got a bug report from a user at https://bugreports.qt.io/browse/QTBUG-106375

_Unfortunately_ Manjaro is shipping qt6ct pre-installed.  They need to stop doing that too, if you don't fix it ASAP.  I can't believe they ship such things without testing anything with Qt Quick.

I didn't dig deeper to see what the actual problem is.

Actually, even qtdiag crashes with this theme plugin.

**Comment by jonaski** [comment]
qt6ct is pretty widely used and part of the repositories in most distros. I've never experienced any crashes with qt6ct. How do you know that the bug is in qt6ct?
[QTBUG-106375](https://bugreports.qt.io/browse/QTBUG-106375) is minor issue with the colors, no mention of crash.
I suggest to do a backtrace of the crash.

**Comment by trialuser02** [comment]
May be qt6ct should be recompiled after Qt upgrade.

**Comment by root2185** [comment]
It completely breaks qt6 here:
```
qml6 --version
Segmentation fault
```
My system:
```
Kernel: 6.4.12-artix1-1 
arch: x86_64 
bits: 64
Distro: Artix Linux
```

**Comment by jonaski** [comment]
None of the utilities crash here.
A rebuild of qt6ct is the first thing you should try, if it still crashes then do a backtrace with gdb and report back.
if Manjaro or Artix Linux didn't do a rebuild of qt6ct after Qt 6 was upgraded, then that's a distro issue.

**Comment by root2185** [comment]
I rebuilt it with asp and now qt crashes again.
After removing it works again.

**Comment by trialuser02** [comment]
Please, test latest commit.

**Comment by root2185** [comment]
It looks fine.

Before I tried to debug with gdb and indeed I saw Palette which I see in the commit:
```
Thread 1 "qml6" received signal SIGSEGV, Segmentation fault.
0x00007ffff69b210a in QGuiApplicationPrivate::setPalette(QPalette const&) () from /usr/lib/qt6/bin/../../libQt6Gui.so.6
```

**Comment by ionenwks** [comment]
Was just trying to package this in Gentoo but ran into the same segfault issues (Qt6.5.2), can confirm the fix seems to sort this out from a quick try, so figure I'll package a snapshot for now.

A release would be appreciated though.

**Comment by root2185** [comment]
I've been using qml fine since the update.

**Comment by trialuser02** [comment]
Fixed in the 0.9 release.
